assembly/Cargo.toml

```diff
@@ -34,5 +34,6 @@ thiserror.workspace = true
 
 [dev-dependencies]
 miden-mast-package = { workspace = true, features = ["arbitrary"] }
+miden-stdlib.workspace = true
```

Comment:
I'm surprised this works, since it forms a circular dependency (`miden-stdlib` depends on `miden-assembly` for both its build script and the code itself). It seems like it works in CI, but that likely has both crates in its build cache, so that may allow it to succeed anyway - does a clean build still succeed?

Reply:
I think because this is a `dev-dependency`, it produces a different node in the dependency graph (iow, `test` produces a different set of binaries). 

As to your last question, you can see this for yourself with :
```
docker run --rm rust:trixie bash -c "apt-get update -qq && apt-get install -y -qq git && git clone --branch issue-2064 https://github.com/huitseeker/miden-vm.git && cd miden-vm && cargo check --all --tests"
```